business/workloads.go

```diff
@@ -508,18 +530,16 @@ func (in *WorkloadService) getParsedLogs(namespace, name string, opts *LogOption
 				startTime = &parsedTimestamp
 			}
 
-			if isBounded {
-				if endTime == nil {
-					end := parsedTimestamp.Add(*opts.Duration)
-					endTime = &end
-				}
+			if endTime == nil {
+				end := parsedTimestamp.Add(*opts.Duration)
+				endTime = &end
+			}
 
-				if parsedTimestamp.After(*endTime) {
-					break
-				}
+			if parsedTimestamp.After(*endTime) {
+				break
 			}
 		} else {
-			log.Debugf("Failed to parse log timestamp (skipping) [%s], %s", entry.Timestamp, err.Error())
+			log.Errorf("Failed to parse log timestamp (skipping) [%s], %s", entry.Timestamp, err.Error())
```

Comment:
Not sure this is a good idea, I think I'd leave this at Debug.

Reply:
I think it's a leftover from a debugging session. I'll revert.